### PR TITLE
Fix reformatter USBs

### PR DESCRIPTION
**Pull request by wjt**
https://phabricator.endlessm.com/T15658

### CODE REVIEW

File: src/endless/EndlessUsbToolDlg.cpp [hunk 1]
```diff
@@ -2413,13 +2413,10 @@ void CEndlessUsbToolDlg::UpdateFileEntries(bool shouldInit)
     // set selected image for Rufus code
```
Comment (ramcq) [inline src/endless/EndlessUsbToolDlg.cpp:2413]: No longer accurate?

**Review by ramcq — APPROVED** [review]
stale comment, otherwise LGTM

File: src/endless/EndlessUsbToolDlg.cpp (hunk 1 above)
Comment (wjt) [reply src/endless/EndlessUsbToolDlg.cpp:2413]: True, will follow up tomorrow

Comment (wjt) [reply src/endless/EndlessUsbToolDlg.cpp:2413]: https://github.com/endlessm/rufus/commit/97d6996bf93b56f4671970b0b8cdd260bdbcab0b

**Pull request merged**